Robolectric unit tests are failing because of ExceptionInInitializerError in mozilla.telemetry.glean.rust.LibGleanFFI

This is happening when running the tests locally on Intel based MacBooks.
Recent CI tasks have all tests running successfully.

Same happens on Focus after the recent android-components update which updates the Glean version - https://github.com/mozilla-mobile/android-components/commit/661ade54ee1b0f865fb5d19076c5b0b3560cfd20

Rolling back to the previous android-components version fixes the problem.

May be linked to https://github.com/mozilla/application-services/issues/4845

&nbsp;
Stacktrace:
```
java.lang.ExceptionInInitializerError
	at mozilla.telemetry.glean.rust.LibGleanFFI.<clinit>(LibGleanFFI.kt)
	at mozilla.telemetry.glean.private.CounterMetricType.<init>(CounterMetricType.kt:42)
	at org.mozilla.fenix.GleanMetrics.PerfStartup.<clinit>(PerfStartup.kt:129)
	at org.mozilla.fenix.FenixApplication.onCreate(FenixApplication.kt:111)
	at org.mozilla.fenix.helpers.FenixRobolectricTestApplication.onCreate(FenixRobolectricTestApplication.kt:20)
	at org.robolectric.android.internal.AndroidTestEnvironment.lambda$installAndCreateApplication$2(AndroidTestEnvironment.java:350)
	at org.robolectric.util.PerfStatsCollector.measure(PerfStatsCollector.java:86)
	at org.robolectric.android.internal.AndroidTestEnvironment.installAndCreateApplication(AndroidTestEnvironment.java:350)
	at org.robolectric.android.internal.AndroidTestEnvironment.lambda$createApplicationSupplier$0(AndroidTestEnvironment.java:229)
	at org.robolectric.util.PerfStatsCollector.measure(PerfStatsCollector.java:53)
	at org.robolectric.android.internal.AndroidTestEnvironment.lambda$createApplicationSupplier$1(AndroidTestEnvironment.java:226)
	at com.google.common.base.Suppliers$NonSerializableMemoizingSupplier.get(Suppliers.java:167)
	at org.robolectric.RuntimeEnvironment.getApplication(RuntimeEnvironment.java:71)
	at org.robolectric.android.internal.AndroidTestEnvironment.setUpApplicationState(AndroidTestEnvironment.java:194)
	at org.robolectric.RobolectricTestRunner.beforeTest(RobolectricTestRunner.java:325)
	at org.robolectric.internal.SandboxTestRunner$2.lambda$evaluate$0(SandboxTestRunner.java:265)
	at org.robolectric.internal.bytecode.Sandbox.lambda$runOnMainThread$0(Sandbox.java:88)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.IllegalStateException: Glean functionality not available
	at mozilla.telemetry.glean.rust.LibGleanFFIKt.loadIndirect$lambda-0(LibGleanFFI.kt:91)
	at com.sun.proxy.$Proxy63.glean_enable_logging(Unknown Source)
	at mozilla.telemetry.glean.rust.LibGleanFFIKt.loadIndirect(LibGleanFFI.kt:96)
	at mozilla.telemetry.glean.rust.LibGleanFFI$Companion.<clinit>(LibGleanFFI.kt:105)
	... 21 more
Caused by: java.lang.UnsatisfiedLinkError: Unable to load library 'xul'
```

--- a/buildSrc/src/main/java/Dependencies.kt
+++ b/buildSrc/src/main/java/Dependencies.kt
@@ -18,7 +18,7 @@ object Versions {
     const val leakcanary = "2.8.1"
     const val osslicenses_plugin = "0.10.4"
     const val detekt = "1.19.0"
-    const val jna = "5.6.0"
+    const val jna = "5.8.0"
 
     const val androidx_activity_compose = "1.4.0"
     const val androidx_compose = "1.0.5"

